docs/os/windows/windows-tools41.md

```diff
@@ -0,0 +1,292 @@
+        
+		[user@untrusted Downloads] $ sha256sum qubes-windows-tools-4.1.67.1.iso
+
+	and compare it to the value stored in the file `sha256sum.txt` for the `iso` file (**it has to exactly match for security reasons**). If it matches, feel free to continue the installation. If not, repeat the download to make sure it was not corrupted due to a network problem. If it keeps on not matching it might be an attacker attempting to do something nasty to your system -- *Ask for support.*
+
+ 4. Start the Windows qube attaching the `iso` file as a virtual CD-Rom drive *(where `<VMname>` is the name of your Windows VM and <AppVM> the name of the AppVM where you downloaded the installation `iso`)*
+
+		[user@dom0 ~] $ qvm-start <VMname> --cdrom=<AppVM>:/<PathToFile>/qubes-windows-tools-4.1.67.1.iso
+
+	Once the Windows VM boots, a CDROM should appear in the 'My Computer' menu (typically as `D:` or `E:`) with the setup program `qubes-tools-x64.msi` in its main directory.
+
+ 5. **Windows 7 only:** Rename the private disk `D:` to `Q:`. This can be done by starting the diskmanager and changing the drive letter:
+	
+	 - Start the command prompt as administrator, i.e. right click on the Command Prompt icon (All Programs -> Accessories) and choose "Run as administrator"
+	 - In the command prompt type `diskmgmt.msc`
+	 - In the disk manager, select the volume `Private (D:)`
+	 - Select the option `Change Drive Letter and Path`
+	 - Select option `Change...`
+	 - Select the letter `Q`
+	 - Click `OK` in all still open windows of the disk manager and terminate it.
+
+ 6. Install Qubes Windows Tools by starting `qubes-tools-x64.msi` as administrator, optionally selecting the `Xen PV disk drivers`. For installation in a template, you should select `Move user profiles`. If during installation, the Xen driver requests a reboot, select "No" and let the installation continue - the system will be rebooted later.
+
+ 7. After successful installation, the Windows VM must be shut down and started again, possibly a couple of times. On each shutdown, wait until the VM is really stopped, i.e. Qubes shows no more activity.
+
+ 8. Qubes will automatically detect that the tools have been installed in the VM and will set appropriate properties for the VM, such as `qrexec_installed`, `guiagent_installed`, and `default_user`. This can be verified (but is not required) using the `qvm-prefs` command  *(where `<VMname>` is the name of your Windows VM)*:
+
+	        [user@dom0 ~] $ qvm-prefs <VMname>
+
+ 	It is advisable to set some other parameters in order to enable audio, synchronize the Windows clock with the Qubes clock, and so on:
+	        
+	        [user@dom0 ~] $ qvm-features <VMname> gui 1
+	        [user@dom0 ~] $ qvm-features <VMname> gui-emulated 1
+	        [user@dom0 ~] $ qvm-features <VMname> audio-model ich9
+	        [user@dom0 ~] $ qvm-features <VMname> stubdom-qrexec 1
+	        [user@dom0 ~] $ qvm-features <VMname> timezone localtime
```

Comment:
timezone feature might be described with more details. With 'localtime' value dom0 time will be provided to virtual hardware. With digit value (negative or positive) the guest clock will have an offset applied relative to UTC (specified in seconds as I remember).

Reply:
O.K. - I'll see to it.

Reply:
I checked - it is in seconds.